browser/src/control/Control.TopToolbar.js

```diff
@@ -137,48 +137,48 @@ L.Control.TopToolbar = L.Control.extend({
 				}, mobile: false},
 			{type: 'break', id: 'breakfontsizes', invisible: true, mobile: false, tablet: false,},
 			{type: 'button', id: 'languagecode', desktop: false, mobile: true, tablet: false},
-			{type: 'button',  id: 'bold',  img: 'bold', hint: _UNO('.uno:Bold'), uno: '.uno:Bold'},
-			{type: 'button',  id: 'italic', img: 'italic', hint: _UNO('.uno:Italic'), uno: '.uno:Italic'},
-			{type: 'button',  id: 'underline',  img: 'underline', hint: _UNO('.uno:Underline'), uno: '.uno:Underline'},
-			{type: 'button',  id: 'strikeout', img: 'strikeout', hint: _UNO('.uno:Strikeout'), uno: '.uno:Strikeout'},
+			{type: 'button', id: 'bold', img: 'bold', hint: _UNO('.uno:Bold'), uno: 'Bold', lockUno: '.uno:Bold'},
```

Comment:
Cases like this are those I'm not happy about - we end up with both uno: 'Bold' and lockUno: '.uno:Bold' ; please let's avoid this :-)